### DISCUSSION TITLE
Zwift not fluid when Group Window is open

### DISCUSSION THREAD

**Discussion by pelleoss** [SauceLLC/sauce4zwift#31] [General]
Hi.
I noticed that the Group Window (the one that shows riders in front/behind you with some statistics) causes my Zwift to be choppy/to stutter... It is enough to close it and  Zwift go back to normal.
My pc is a i5-7300U with 8GB RAM.
Can the cause be the pc? Is anybody facing the same issue?
Can I set something in order to avoid it?
The Group window is the one I like the most but watching Zwift stuttering and glitching is really annoying.
Thanks!

**Comment by mayfield** [comment]
Hi @pelleoss

Most likely your system is just getting a little overloaded.  In the future I might add a low power mode that will ease up the visual animations that are the source of most of the CPU usage.  In the mean time you might be able to lower the graphics settings of Zwift a bit or use TaskManager to change the prio of zwift to be higher (although I've not tested this).

**Comment by suskozaver** [comment]
@pelleoss what kind of graphic card are you running? in what mode are you running Zwift? what's the graphic detail? try lowering it, so the graphic power is distributed between Zwift & Sauce.
